client/src/tests/e2e/datapass.spec.ts

```diff
@@ -21,4 +22,63 @@ test.describe("Datapass", () => {
 
     await page.locator("text='Recherchez un jeu de données'").waitFor();
   });
+
+  test("A user can NOT pick the organization he wants to be linked with -- invalid token case", async ({
+    page,
+  }) => {
+    const token = "not-valid";
```

Comment:
Je m'étais demandé où est-ce qu'on testait le "cas nominal" ("si tout se passe bien")

En fait, c'est ici ! Mais on s'attend à obtenir une 403.

Je pense qu'il faudrait juste mettre un commentaire sur le fait qu'on s'attend à une 403 parce qu'obtenir une vraie 201 du back serait compliqué (il faudrait obtenir un token valide ce qui n'est pas possible sans un mocking complexe de DataPass).

```suggestion
  test("A user picks the organization they want to be linked with", async ({
    page,
  }) => {
    const token = "<fake>";
```